lib/kb_plant_rast/kb_plant_rastImpl.py

```diff
@@ -153,22 +168,32 @@ def annotate_plant_transcripts(self, ctx, input):
         output['hit_ftrs']=len(Hit_Proteins)
         output['hit_fns']=len(Hit_Functions)
 
+        #But, if annotating CDS, need to be able to retrieve parent feature
+        parent_feature_index = dict()
+        if(USE_CDS==1):
+            for i in range(len(plant_genome['data']['features'])):
```

Comment:
One line option: parent_feature_index = dict([(f['id'], i) for i, f in plant_genome['data']['features']])